libcontainer/configs/validate/validator.go

```diff
@@ -128,6 +128,41 @@ func (v *ConfigValidator) cgroupnamespace(config *configs.Config) error {
 	return nil
 }
 
+// convertSysctlVariableToDotsSeparator can return sysctl variables in dots separator format.
+// The '/' separator is also accepted in place of a '.'.
+// Convert the sysctl variables to dots separator format for validation.
+// More info:
+//   https://man7.org/linux/man-pages/man8/sysctl.8.html
+//   https://man7.org/linux/man-pages/man5/sysctl.d.5.html
+func convertSysctlVariableToDotsSeparator(val string) string {
```

Comment:
Could you add example inputs and outputs the code comment?
Also could you consider adding unit tests

Reply:
I can add unit tests, but this package test uses `package validate_test`.
Then this function must be exported, otherwise it cannot be referenced. I am not sure whether to export this function, after all, this method is only used in this package.

Reply:
The package name should be renamed to `validate`, then you can add unit tests without exporting the function.

Reply:
Okay, I have renamed the test package name to `validate`.